libs/src/services/ethContracts/index.js

```diff
@@ -61,11 +61,21 @@ class EthContracts {
     )
     this.getRegistryAddressForContract = this.getRegistryAddressForContract.bind(this)
 
+    this.StakingProxyClient = new StakingProxyClient(
```

Comment:
had to move staking proxy client above other client initialization steps. this needed to be passed into GovernanceClient

Reply:
weird that it was already a parameter but this makes sense